node/test/Test/Pos/CborSpec.hs

```diff
@@ -223,11 +232,12 @@ roundtripProperty (input :: a) = ((deserialize . serialize $ input) :: a) === in
 -- without breaking anything. This should work with every time which adopted
 -- the schema of having at least one constructor of the form:
 -- .... | Unknown Word8 ByteString
-extensionProperty :: forall a. (Arbitrary a, Eq a, Show a, Bi a) => Property
-extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
+extensionPropertyOn :: forall a b. (Eq a, Show a, Bi a, Arbitrary b, Eq b, Show b, Bi b)
+                    => Property
+extensionPropertyOn = forAll @b (arbitrary :: Gen b) $ \input ->
```

Comment:
Something which I _think_ we don't need (and I think it's a relic) is the need for the "double" set of constraints (apart from the one on `Bi b`) . We just need to generate the the input type, not `U24`, thus we can simplify the signature to:

```
extensionPropertyOn :: forall a b. (Bi a, Arbitrary b, Eq b, Show b, Bi b) => Property
```